test/commands/commit.spec.js

```diff
@@ -157,9 +159,30 @@ describe('commit command', () => {
       })
     })
 
+    describe('when rebasing', () => {
+      it('should cancel the hook', async () => {
+        // Use an exception to suspend code execution to simulate process.exit
+        mockProcess.mockProcessExit(new Error('ProcessExit0'))
```

Comment:
Why we can't simulate process.exit here ? 🤔

Reply:
I'm simulating process exit by using an exception, if process exit 1 happen during the test, the test is killed, that's why I mock the process exit. I'm using an exception to interrupt as well the code execution to further mimic a process exit